lib/modules/dosomething/dosomething_api/resources/reportback_resource.inc

```diff
@@ -133,14 +154,12 @@ function _reportback_resource_index($campaigns, $status, $count, $random, $page,
     'page' => $page,
     'load_user' => $load_user,
     'flagged' => $flagged,
-  );
+  ];
 
-  $reportbacks = new ReportbackTransformer;
-  return $reportbacks->index($parameters);
+  return (new ReportbackTransformer)->index($parameters);
```

Comment:
I changed it cause I have it like that in the campaigns resource and else where too... if it's not as readable I can change the resources globally. Don't really have a preference other than making all match :)